src/server/scripts/Commands/cs_reset.cpp

```diff
@@ -36,10 +36,27 @@ using namespace Acore::ChatCommands;
 class reset_commandscript : public CommandScript
 {
 public:
+
+    enum
+    {
+        ITEM_EARTH_STONE = 6948, // Perhaps should be defined in a more "common" file if another piece of code needs it
```

Comment:
```suggestion
        ITEM_HEARTHSTONE = 6948, // Perhaps should be defined in a more "common" file if another piece of code needs it
```

Reply:
Why prevent specifically the hearthstone from being deleted? It can be deleted like any other item to a player, and isn't hard to receive again, especially with other commands.

Reply:
Hum good question.
You mean it could be removed without any problem ?

Reply:
I see no reason why not.